### PR TITLE
Added list-plans command.

### PR DESCRIPTION
**Pull request by alesstimec**

### CODE REVIEW

File: cmd/list.go [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+import (
+	"strings"
+
+	"github.com/juju/cmd"
+	"github.com/juju/errors"
+	"github.com/juju/gnuflag"
+)
+
+const listPlansDoc = `
+list-plans is to list plans
```
Comment (mattyw) [inline cmd/list.go]: list-plans lists all the plans owned by the specified user

File: api/wireformat/entities.go [hunk 2]
```diff
@@ -51,15 +51,15 @@ type PlanActive struct {
...
-	URL             string      `json:"url"`        // Name of the rating plan
-	Definition      string      `json:"plan"`       // The rating plan source
-	CreatedOn       string      `json:"created-on"` // When the plan was created - RFC3339 encoded timestamp
-	PlanDescription string      `json:"description"`
-	PlanPrice       string      `json:"price"`
-	Released        bool        `json:"released"`
-	EffectiveTime   *time.Time  `json:"effective-time,omitempty"`
-	Model           interface{} `json:"model,omitempty"` // The rating plan model
+	Id              string      `json:"id" yaml:"id"`                 // Full id of the plan format
+	URL             string      `json:"url" yaml:"url"`               // Name of the rating plan
```
Comment (tasdomas) [inline api/wireformat/entities.go]: URL is part of the Id, right? If so, do we need to output it?

Comment (alesstimec) [reply api/wireformat/entities.go]: this was not added in this PR..

File: cmd/list.go (hunk 1 above)
Comment (alesstimec) [reply cmd/list.go]: @mattyw it might be a group, not a user.. like.. canonical..

Comment (cmars) [inline cmd/list.go]: How about: `list plans owned by a user or group`

**Comment by alesstimec** [comment]
$$merge$$

**Pull request merged**